routes/api/gameRoutes.js

```diff
@@ -0,0 +1,35 @@
+const Games = require("../../models/Games");
+const { Op } = require("sequelize");
+
+const router = require("express").Router();
+
+// create new game
+router.post("/", async (req, res) => {
+  const { gameTitle } = req.body;
+
+  if (
+    ![gameTitle].every(
```

Comment:
no point in doing it this way as it's a single value.

this should be fine.
```suggestion
    if(!gameTitle.trim()) {}
```

Reply:
Yeh, sorry my bad!

Reply:
fixing it now